packages/components/src/toggle-group-control/types.ts

```diff
@@ -98,17 +98,17 @@ export type ToggleGroupControlProps = Pick<
 	 */
 	isBlock?: boolean;
 	/**
-	 * Borderless style that may be preferred in some contexts.
+	 * Whether an option can be deselected by clicking it again.
 	 *
 	 * @default false
 	 */
-	__experimentalIsBorderless?: boolean;
```

Comment:
Given the recent conversations around deprecating and removing props, are we able to simply delete this prop, even if it's experimental?

Reply:
Until any new rules are codified, I'd say yes. It also helps that this is a cosmetic difference that doesn't break layout.